lib/MusicBrainz/Server/EditSearch/Predicate/EditNoteContent.pm

```diff
@@ -0,0 +1,35 @@
+package MusicBrainz::Server::EditSearch::Predicate::EditNoteContent;
+use Moose;
+
+with 'MusicBrainz::Server::EditSearch::Predicate';
+
+sub operator_cardinality_map {
+  return (
+      'includes' => undef,
+  )
+}
+
+sub combine_with_query {
+    my ($self, $query) = @_;
+
+    $query->add_where([
+        'EXISTS (
+          SELECT TRUE FROM edit_note
+          WHERE edit_note.text LIKE ?
+            AND edit_note.edit = edit.id
+        )',
+        [ map { '%' . $_ . '%' } @{ $self->sql_arguments } ]
```

Comment:
You'll have to escape `%` and `_` and `\` there since those three chars have special meaning. Maybe something like
```perl
    my @patterns = map {
      $_ =~ s/\\/\\\\/g;
      $_ =~ s/_/\\_/g;
      $_ =~ s/%/\\%/g;
      '%' . $_ . '%'
    } @{ $self->sql_arguments };
```

```suggestion
        \@patterns,
```